services/security/identity/identity.go

```diff
@@ -113,6 +110,17 @@ func getWssdIdentity(id *security.Identity) (*wssdcloudsecurity.Identity, error)
 
 	wssdidentity.ClientType = clitype
 	wssdidentity.AutoRotate = id.AutoRotate
+
+	// In case auto rotate in enabled, the login file path can not be empty or relatively
+	if id.AutoRotate == true && id.LoginFilePath != nil {
+		if *id.LoginFilePath == "" {
+			return nil, errors.Wrapf(errors.InvalidInput, "Identity Loginfile path cannot be empty")
```

Comment:
This should not return a error. Why are we returning a error here?

Reply:
Are we not writing to the location of LoginFilePath if we have enabled token auto rotation?

Reply:
No, when autorotate is enabled, the file path can be empty or can be absolute path. It is not necessary for the path to be there for autorotate

Reply:
Understood, Done